### ISSUE TITLE
[RyuJIT/ARM32] codegenarm.cpp:294 - NYI_ARM: struct return

### ISSUE THREAD

**Issue by hseok-oh** [dotnet/coreclr#12451]
Example 

```
Assert failure(PID 13226 [0x000033aa], Thread: 13226 [0x33aa]): Assertion failed 'NYI_ARM: struct return' in 'hfatest.BringUpTest:Get_HFA02():struct' (IL size 9)

    File: /nfs/rbp3/coreclr/src/jit/codegenarm.cpp Line: 294
    Image: /nfs/rbp3/test/coreoverlay_checked_arm/corerun

```

Test methods with this NYI message
```
<>c:<.cctor>b__3_0(struct):struct:this
<>c:<.cctor>b__3_1(struct):struct:this
<>c:<.cctor>b__3_3(struct):struct:this
<>c:<.cctor>b__3_4(struct):struct:this
<>c:<.cctor>b__3_6(struct):struct:this
<>c:<.cctor>b__3_7(struct):struct:this
DomainBoundILStubClass:IL_STUB_PInvoke(ref,struct):struct
DomainBoundILStubClass:IL_STUB_PInvoke():struct
DomainBoundILStubClass:IL_STUB_PInvoke(struct):struct
HFATest.TestMan:Get_HFA02():struct
HFATest.TestMan:Get_HFA03():struct
HFATest.TestMan:Identity_HFA02(struct):struct
HFATest.TestMan:Identity_HFA03(struct):struct
Plane:Normal(struct):struct:this
RayTracer:GetPoint(double,double,ref):struct:this
RayTracer:Shade(ref,ref,int):struct:this
Sphere:Normal(struct):struct:this
System.Numerics.Vector3:get_Zero():struct
System.Numerics.Vector3:Normalize(struct):struct
System.Numerics.Vector3:op_Addition(struct,struct):struct
System.Numerics.Vector3:op_Division(struct,float):struct
System.Numerics.Vector3:op_Multiply(float,struct):struct
System.Numerics.Vector3:op_Multiply(struct,struct):struct
System.Numerics.Vector3:op_Subtraction(struct,struct):struct
Test01.Program:RandomSimpleVector3():struct
Test01.Program:RandomVector3():struct
Test01.Program:RandomWrappedVector3():struct
Test.Program:getTestValue(float,float,float):struct
```

**Comment by hseok-oh** [comment]
I'm going to look this issue.
It looks like related to HFA return.

**Comment by hseok-oh** [comment]
Fixed by #12453
